cypress/e2e/notifications.spec.ts

```diff
@@ -0,0 +1,112 @@
+import * as api from "../api"
+import {
+  E2E_EMAIL_ADMIN,
+  E2E_EMAIL_COLLAB,
+  E2E_EMAIL_TEST_SITE,
+} from "../fixtures/constants"
+import {
+  genRandomString,
+  getNotificationsButton,
+  visitE2eEmailTestRepo,
+} from "../utils"
+
+const NEW_COMMENTS_BG_COLOR = "rgb(235, 248, 255)"
```

Comment:
is _unread_ comments rather than new comments a better phrasing here?

Reply:
good point, changed